engine/Components/CreateConditionals.py

```diff
@@ -0,0 +1,36 @@
+
+
+def create_conditionals(statement):
+	"""
+	Creates the conditional -> from ['-', '>'] and
+	biconditional <-> from ['<', '-', '>'].
+
+	:param statement: takes a statement with individual pieces in an array.
+	:return: Returns a new array with connectives as one element.
+	"""
+
+	elements = []
+	conditional_symbol = False
+	biconditional_symbol = False
+	for i in range(len(statement)):
+		char = statement[i]
+		if char == '<':
```

Comment:
Wouldn't it be easier to match fast than keep branching? For example, 

```python
if statement[i] == '<' && statement[i+1] == '-' && statement[i+2] == '>' ...
```
than to branch into an if statement, and then branch again and branch again. A negations statement might also work, and possibly even a fail first check before branching.

Reply:
```
if statement[i] != '-':
   // continue

elif statement[i+1] == '>' && statement[i-1] != '<':
   // conditional
elif  statement[i+1] == '>' && statement[i-1] == '<':
// biconditional
else
// error

Reply:
Updated.